sync/src/main/java/org/bf2/sync/informer/InformerManager.java

```diff
@@ -46,6 +51,19 @@ protected void onStart() {
         managedAgentInformer = resourceInformerFactory.create(ManagedKafkaAgent.class, client.resources(ManagedKafkaAgent.class).inAnyNamespace(),
                 CustomResourceEventHandler.of(controlPlane::updateAgentStatus));
 
+        secretInformer = resourceInformerFactory.create(Secret.class, client.secrets().inAnyNamespace().withLabels(OperandUtils.getMasterSecretLabel()),
+                new ResourceEventHandler<Secret>() {
```

Comment:
This can just be null.

Reply:
Do you mean the inside the call for the new `ResourceEventHandler` or the type of `ResourceEventHandler` ?